File: 021-Absolute Difference/AbsoluteDifference.py
def findCount(arr, num, diff):
    count = 0
    for val in arr:
        if abs(val - num) <= diff:
            count += 1
    return count if count > 0 else -1

# Sample usage
arr = [12, 3, 14, 56, 77, 13]
num = 13
diff = 2
print(findCount(arr, num, diff))

File: 021-Absolute Difference/AbsoluteDifference.cpp
#include<iostream>
#include<cmath>
using namespace std;

int findCount(int arr[], int length, int num, int diff){
    int count = 0;
    for(int i = 0; i < length; i++){
        if(abs(arr[i] - num) <= diff){
            count++;
        }
    }

    if(count == 0){
        return -1;
    }else{
        return count;
    }
}
int main(){
    int arr[] = {12,3,14,56,77,13};
    int length = sizeof(arr)/ sizeof(arr[0]);
    int num = 13;
    int diff = 2;

    cout<<findCount(arr,length,num,diff)<<endl;

    return 0;
}